independent-publisher-connectors/Zippopotamus/apiDefinition.swagger.json

```diff
@@ -0,0 +1,375 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zippopotamus",
+    "description": "API allows to obtain location details for a given postal code or state and city. It provides data for over 60 countries.",
```

Comment:
@tposzytek Can you please add the product/service details to the description as it seems API centric explanation?

Reply:
Is this somehow closer to what you were expecting?